packages/lab/src/__tests__/__e2e__/date-input/DateInput.cy.tsx

```diff
@@ -0,0 +1,51 @@
+import { DateInput } from "@salt-ds/lab";
+import { ChangeEvent } from "react";
+
+describe("GIVEN an DateInput", () => {
+  it("SHOULD have no a11y violations on load", () => {
+    cy.mount(<DateInput defaultValue="01 Feb 2000" />);
+    cy.checkAxeComponent();
```

Comment:
is this doing the same as our checkAccessibility() function? just wondering why sometimes we do one or the other

Reply:
oh yeah, i think this predates the other one and i copied the scaffold from an older component, will update it